postgres-appliance/major_upgrade/inplace_upgrade.py

```diff
@@ -172,13 +172,15 @@ def ensure_replicas_state(self, cluster):
         """
         self.replica_connections = {}
         streaming = {a: l for a, l in self.postgresql.query(
-            ("SELECT client_addr, pg_catalog.pg_{0}_{1}_diff(pg_catalog.pg_current_{0}_{1}(),"
+            ("SELECT client_addr, application_name, pg_catalog.pg_{0}_{1}_diff(pg_catalog.pg_current_{0}_{1}(),"
```

Comment:
This does not work. The result it put in a dictionary, so it expects to unpack two elements, not three. You need to select either `client_addr` or `application_name`.